Sources/Player/PlaybackEngine/Internal/Events/PlayerEventSender.swift

```diff
@@ -53,12 +53,14 @@ class PlayerEventSender {
 		if let eventsDirectory = PlayerEventSender.initializeDirectory(name: "player_events") {
 			self.eventsDirectory = eventsDirectory
 		} else {
+			PlayerWorld.logger?.log(loggable: PlayerLoggable.eventSenderInitEventsDirectoryFailed)
```

Comment:
App will crash on the next line, not sure if this will be logged! 🤔

Reply:
Yes, I saw it so was not sure, but added just in case.